dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -261,6 +268,21 @@ private void handlePotentialFormatEntry(Set<FileFormat> fileFormats, Object v) {
 
             // Convert YAML to JSON
             Map<String, Object> mapping = yaml.loadAs(mainDescriptor, Map.class);
+
+            // verify cwl version is correctly specified
+            final Object cwlVersion = mapping.get("cwlVersion");
+            if (cwlVersion != null) {
+                final boolean equals = cwlVersion.toString().startsWith(CWLHandler.CWL_VERSION_PREFIX);
```

Comment:
The variable name is `equals`, but you're not testing for equality, you're testing for `startsWith`. A little confusing, and I'm not sure which one you want.

Nvm, I see it was like this already.

Reply:
Modified both areas to have a more descriptive variable name, see [here](https://github.com/dockstore/dockstore/blob/76d246baa2dabe6b004914606056c5645a7acb12/dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java#L275).